calculator.rb

```diff
@@ -0,0 +1,75 @@
+def add(num_1, num_2)
+  return  "\n#{num_1} + #{num_2} = #{num_1 + num_2} "
+end
+def substract(num_1, num_2)
+  return "\n#{num_1} - #{num_2} = #{num_1 - num_2} "
+end
+def multiply(num_1, num_2)
+  return "\n#{num_1} * #{num_2} = #{num_1 * num_2} "
+end
+def divide(num_1, num_2)
+  return "\n#{num_1} / #{num_2} = #{num_1 / num_2} "
+end
+
+operator_array = ["add","+", "substract","-", "multiply","*", "divide","/"]
+puts  "\nWelcome to the Calculator Program!"
+print "How many operations would you like to do?: " 
+operator_num = gets.chomp
+
+while operator_num !~ /^-?[0-9]+$/
+  puts "NUMBERS ONLY!!!!!"
+  print "\nHow many operations would you like to do?: " 
+  operator_num = gets.chomp
+end 
+operator_num = operator_num.to_i
+
+operator_num.times do |i|
```

Comment:
This is a very nice feature. I enjoyed being able to complete multiple problems without rerunning the program!